src/styles/theme.scss

```diff
@@ -169,16 +188,28 @@ h5 {
   }
 }
 
+.link[href*="#/stories"] {
+  color: $primary-colour;
+}
+
 /* ------------------------------
 
    Button
 
 ------------------------------ */
 
+.section .button-group {
+  display: flex;
```

Comment:
We should not need to alter any layout stuff

Reply:
well, it seems to me that Holly's design was departing a bit from the core layout. better to test it out with this to see the difference.

Reply:
I still prefer to take all of this out. Altering the layout opens us up for undetected visual bugs. We designed it as much as possible to be used with just changing colours and fonts.

Reply:
I understand that but Holly told me that's what the seAp team wanted. If we do this, then why make the buttons rectangular? Just as an example. Their design is to a large extent ignored... but I can do whatever you ask me. Just at this point I feel a bit confused as to what my supervisor tells me and what the client's design is indicating :-)